airflow/config_templates/default_celery.py

```diff
@@ -31,6 +30,9 @@
     'broker_transport_options': {'visibility_timeout': 21600},
     'result_backend': configuration.get('celery', 'CELERY_RESULT_BACKEND'),
     'worker_concurrency': configuration.getint('celery', 'CELERYD_CONCURRENCY'),
+    'database_engine_options': {
```

Comment:
maybe we don't want this .. but it is the mvp lol

Reply:
by this I mean the database engine options in the default config. although it should be OUR default config